diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9eb550f..b58e3f0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,17 +4,15 @@ if(COMMAND cmake_policy)
 endif(COMMAND cmake_policy)
 
 project(CNPY)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
 option(ENABLE_STATIC "Build static (.a) library" ON)
 
-find_package(ZLIB REQUIRED)
-
-include_directories(${ZLIB_INCLUDE_DIRS})
-
 add_library(cnpy SHARED "cnpy.cpp")
-target_link_libraries(cnpy ${ZLIB_LIBRARIES})
+target_link_libraries(cnpy ${CONAN_LIBS})
 install(TARGETS "cnpy" LIBRARY DESTINATION lib PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 
 if(ENABLE_STATIC)
